scripts/scrape-station-names.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console */
+/* eslint-disable no-plusplus */
+
+const fs = require('fs');
+const fetch = require('isomorphic-fetch');
+
+let count = 0;
+const APIKEY = '';
+const PORTION = 9;
+
+const stations = JSON.parse(fs.readFileSync('../data/station-en.json')).slice(PORTION * 1000, (PORTION + 1) * 1000);
+
+// Using timeout (fast and does not exceed rate-limit)
+const func = async (station) => {
```

Comment:
Can you give this a real name besides func?

Reply:
Fixed!